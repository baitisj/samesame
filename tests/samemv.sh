NAME=samemv
PARAM=move
. ${srcdir}/tests/include_action.sh
