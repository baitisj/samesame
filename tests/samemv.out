samemv -A:
./work/test1/file2a
./work/test1/file3a

samemv -Z:
./work/test1/file2a
./work/test1/file1a

samemv -At:
./work/test1/file3a
./work/test1/file2a

samemv -Zt:
./work/test1/file3a
./work/test1/file1a

samemv -L:
./work/test1/file1a
./work/test1/file3a
